test/fixtures/data.js

```diff
@@ -89,37 +87,18 @@ const dataWithNullAndUndefined = [
   },
 ];
 
-export const dataArrayWithHeader = [
-  ...headerArray,
-  ...dataArray,
-];
+exports.dataArrayWithHeader = [...headerArray, ...dataArray];
```

Comment:
Why did you change the way of exporting variables?

Reply:
Yep that I could see, but once you do `export.variableName`, then `const variableName = ''` and `module.exports = { variableName };`.

Jest can run test with `import` and `export` if using babel. I would prefer it that way.